content/concepts/target-market-settings.md

```diff
@@ -171,9 +187,106 @@ Below is the JSON standard format representing a measure family.
 }
 ```
 
-::: warning
-Endpoints for the measure families are only available starting the 2.0 version.
+::: panel-link Want more details about the measure family resource? [Check its endpoints here!](/api-reference.html#Measurefamily)
 :::
 
-::: panel-link Want more details about the measure family resource? [Check its endpoints here!](/api-reference.html#Measurefamily)
+## Measurement family
+::: availability versions=Serenity editions=CE,EE
 :::
+
+If you want to store your product measurement, i.e. weight, height or power inside your PIM, you will need measurement families. These entities will be really helpful when you are requesting products for a given channel and you want these measurement attributes to be converted into the units you set in the given channel.
+
+Below is an example of one of these measurement attributes.
+
+![Metrics attribute](/img/concepts/metrics_attributes_ui.png)
+
+Below is the JSON standard format representing a measurement family.
+
+```json
+{
+  "code": "AREA",
+  "labels": {
+    "en_US": "Area",
+    "fr_FR": "Surface"
+  },
+  "standard_unit_code": "SQUARE_METER",
+  "units":[
+    {
+      "code": "SQUARE_MILLIMETER",
+      "labels": {
+        "en_US": "Square millimeter",
+        "fr_FR": "Millimètre carré"
+      },
+      "convert_from_standard": [
+        {
+          "operator": "mul",
+          "value": "0.000001"
+        }
+      ],
+      "symbol": "mm²"
+    },{
+      "code": "SQUARE_CENTIMETER",
+      "labels": {
+        "en_US": "Square centimeter",
+        "fr_FR": "Centimètre carré"
+      },
+      "convert_from_standard": [
+        {
+          "operator": "mul",
+          "value": "0.0001"
+        }
+      ],
+      "symbol": "cm²"
+    },{
+      "code": "SQUARE_METER",
+      "labels": {
+        "en_US": "Square meter",
+        "fr_FR": "Mètre carré"
+      },
+      "convert_from_standard": [
+        {
+          "operator": "mul",
+          "value": "1"
+        }
+      ],
+      "symbol": "m²"
+    },...
+  ]
+}
+```
+::: info
+You can have at max 100 measurements families and 50 units per measurement family.
```

Comment:
```suggestion
You can have at max 100 measurements families and 50 units per measurement family.  
```
You need to add a double space at the end of your line to force a going to the next line.